src/main/PileupTrack.js

```diff
@@ -212,6 +212,8 @@ class PileupTrack extends React.Component {
   cache: PileupCache;
   tiles: TiledCanvas;
   static defaultOptions: { viewAsPairs: boolean };
+  static getOptionsMenu: (options: Object)=>any;
```

Comment:
Only because I know you're a fan of consistency, but space after `=>` (as is the case below), or not?